### ISSUE TITLE
[BUG] Incorrect filters for SMRT Home page, other filter changes

### ISSUE THREAD

**Issue by dianamartin** [cityofaustin/atd-data-tech#7970]
Andrea M
SMS

Messaged
>Just fyi.. I don't think the Home Page on the SMRT Tracker is working correctly. The number of devices is off and the expiring insurance isn't showing up

**Comment by dianamartin** [comment]
cc @patrickm02L 
I called Andrea to sync with her about the issue she brought to my attention
>The number of devices is off and the expiring insurance isn't showing up

I temporarily added an expanded group view to a pivot table for (Active Permits), we discovered a weird permit for Ojo. 
- [x] Deleted 2 permits for OJO that were testing leftovers that I created 

Took a look at the filters on the Home page for expiring records. The filters seemed off, so.. 
- [x] I updated them from `from next 30 days` to `during the current - quarter`. Changing this showed 2 insurances that were supposed to be due to Andrea's knowledge.
- [x] Update the description on each table to say it's for the next quarter and not the next 30 days.

- [x] Add Filters to Invoices: 
- PAID
- UNPAID
- PENDING
They need to determine the difference between Unpaid and Pending statuses
- Change blank invoice status to "Pending"
